airflow/models/taskinstance.py

```diff
@@ -382,13 +382,6 @@ def generate_command(dag_id: str,
             cmd.extend(["--cfg-path", cfg_path])
         return cmd
 
-    @property
-    def log_filepath(self):
-        iso = self.execution_date.isoformat()
-        log = os.path.expanduser(conf.get('logging', 'BASE_LOG_FOLDER'))
-        return ("{log}/{dag_id}/{task_id}/{iso}.log".format(
-            log=log, dag_id=self.dag_id, task_id=self.task_id, iso=iso))
-
```

Comment:
We should still support it, we can check if we are using a remote_logging or not.

Let's not remove this - I know of many people/companies that are using this property.